watertap/unit_models/tests/test_coag_floc_model.py

```diff
@@ -104,19 +104,50 @@ def test_build_model(self, coag_obj_w_chems):
         assert isinstance(model.fs.unit.tss_loss_rate, Var)
         assert isinstance(model.fs.unit.eq_tss_loss_rate, Constraint)
 
+        assert isinstance(model.fs.unit.rapid_mixing_retention_time, Var)
+        assert isinstance(model.fs.unit.num_rapid_mixing_basins, Var)
+        assert isinstance(model.fs.unit.rapid_mixing_vel_grad, Var)
+
+        assert isinstance(model.fs.unit.floc_retention_time, Var)
+        assert isinstance(model.fs.unit.single_paddle_length, Var)
+        assert isinstance(model.fs.unit.single_paddle_width, Var)
+        assert isinstance(model.fs.unit.paddle_rotational_speed, Var)
+        assert isinstance(model.fs.unit.paddle_drag_coef, Var)
+        assert isinstance(model.fs.unit.vel_fraction, Var)
+        assert isinstance(model.fs.unit.num_paddle_wheels, Var)
+        assert isinstance(model.fs.unit.num_paddles_per_wheel, Var)
+
         assert isinstance(model.fs.unit.tds_gain_rate, Var)
         assert isinstance(model.fs.unit.eq_tds_gain_rate, Constraint)
 
         assert isinstance(model.fs.unit.eq_mass_transfer_term, Constraint)
 
+        assert isinstance(model.fs.unit.eq_rapid_mixing_basin_vol, Constraint)
+        assert isinstance(model.fs.unit.rapid_mixing_basin_vol, Var)
+
+        assert isinstance(model.fs.unit.eq_rapid_mixing_power, Constraint)
+        assert isinstance(model.fs.unit.rapid_mixing_power, Var)
+
+        assert isinstance(model.fs.unit.eq_floc_basin_vol, Constraint)
+        assert isinstance(model.fs.unit.floc_basin_vol, Var)
+
+        assert isinstance(model.fs.unit.eq_floc_wheel_speed, Constraint)
+        assert isinstance(model.fs.unit.floc_wheel_speed, Var)
+
+        assert isinstance(model.fs.unit.eq_flocculation_power, Constraint)
+        assert isinstance(model.fs.unit.flocculation_power, Var)
+
+        assert isinstance(model.fs.unit.eq_total_power, Constraint)
+        assert isinstance(model.fs.unit.total_power, Var)
+
     @pytest.mark.unit
     def test_stats(self, coag_obj_w_chems):
         model = coag_obj_w_chems
 
         # Check to make sure we have the correct DOF and
         #   check to make sure the units are correct
         assert_units_consistent(model)
-        assert degrees_of_freedom(model) == 11
+        assert degrees_of_freedom(model) == 11 + 11
```

Comment:
Why not just do 22? At most, I would add a comment explaining what the difernt DoF are, but just test against the total number,

I see this in a few other places below too.

Reply:
Updated all asserts based on suggestion.